include/modem/trace_backend.h

```diff
@@ -40,11 +40,8 @@ struct nrf_modem_lib_trace_backend {
 
 	/**
 	 * @brief Deinitialize the compile-time selected trace backend.
-	 *
-	 * @return 0 If the operation was successful.
-	 *         Otherwise, a (negative) error code is returned.
 	 */
-	int (*deinit)(void);
+	void (*deinit)(void);
```

Comment:
I think it's best to keep it returning an `int`, even if current implementations can't fail. That is still a possibility if they change or new ones are added; by removing the return value and the log it's impossible to tell if this operation failed.

Reply:
updated